uftrace.h

```diff
@@ -419,6 +419,8 @@ enum uftrace_msg_type {
 	UFTRACE_MSG_SEND_END,
 
 	UFTRACE_MSG_AGENT_CLOSE = 200, /* close the connection */
+	UFTRACE_MSG_AGENT_OK, /* ack previous message */
+	UFTRACE_MSG_AGENT_ERR, /* signal error on previous message */
```

Comment:
We can just have `UFTRACE_MSG_AGENT_RESP` and pass 0 for OK, non-0 for ERR.  That'd simplify handling responses a bit.

Reply:
I'm using the OK response to carry data. For example, on a query, the capabilities are returned as the data load of the OK message. Should I modify this?

> ```
>       client                              agent
> -------------------            ----------------------------
> QUERY []            --->
>                           <--- OK [features (DEPTH | TIME)]
>
> SET_OPT [DEPTH, 3]  --->
>                           <--- OK []
>
> SET_OPT [SIZE, 100] --->
>                           <--- ERR [NOT SUPPORTED]
> 
> CLOSE []            --->
>                           <--- OK []
> ```